packages/babel-runtime-corejs2/helpers/esm/iterableToArray.js

```diff
@@ -1,5 +1,9 @@
 import _Array$from from "../../core-js/array/from";
+import _Set from "../../core-js/set";
+import _Map from "../../core-js/map";
```

Comment:
It's a problem for runtime - dozens of kilobytes of useless dependencies.

Reply:
Hmm - these should be using the global, and not pulling from core-js at all. How can we prevent the build process from incorrectly transpiling these?